Fix unintended control transfers in webserial example.

ipopov opened the pull request:
As discussed in https://github.com/hathach/tinyusb/issues/609,

The problem is that when receiving a "standard" CLEAR_FEATURE request, usbd.c calls into the _class_ driver:
https://github.com/hathach/tinyusb/blob/a96ee8f1d850bbd645040e0f5d133e6c1c93a3a4/src/device/usbd.c#L757. (The comment there says that the class *must not* call tud_control_status.)

Unfortunately the "class driver", i.e. the webusb example's main function, intereprets this as a "class" request with bRequest==1 (i.e. VENDOR_REQUEST_WEBUSB), rather than as a "endpoint" request with bRequest==CLEAR_FEATURE,
https://github.com/hathach/tinyusb/blob/a96ee8f1d850bbd645040e0f5d133e6c1c93a3a4/examples/device/webusb_serial/src/main.c#L156, and sends back a report.

To fix this, the class driver should ignore requests that are not clearly meant for it.

hathach commented on examples/device/webusb_serial/src/main.c:
```diff
@@ -151,52 +151,47 @@ bool tud_vendor_control_xfer_cb(uint8_t rhport, uint8_t stage, tusb_control_requ
+          return tud_control_xfer(rhport, request, (void*) desc_ms_os_20, total_len);
+        } else {
+          return false;
+        }
```
all switch should have `default`, please add one with return false;

ipopov replied:
Done. :-)

hathach reviewed (changes requested):
Thank you very much for the PR, this is spot-on. I am too lazy to check for the request vendor type. There is only a couple of style feedback
1. Please add `default: return false;` to the switch(bRequest)
2. I am not against openning `{` in the same line with new code, however, you should not reformat the existing style.

ipopov commented:
Thank you for the kind comments! It's a wonderful codebase, I had a great time this weekend getting to know it. I hope I can be of assistance in the future as well.

I've uploaded a new commit addressing your comments.

hathach commented on examples/device/webusb_serial/src/main.c:
```diff
@@ -151,52 +151,54 @@ bool tud_vendor_control_xfer_cb(uint8_t rhport, uint8_t stage, tusb_control_requ
+    web_serial_connected = (request->wValue != 0);
+
+    // Always lit LED if connected
+    if ( web_serial_connected ) {
```
not enirely with braket, but this is OK

hathach reviewed (approved):
Thank you very much for your PR. I am appreciate that you have spent time to troubleshoot this issue.

hathach commented:
> Thank you for the kind comments! It's a wonderful codebase, I had a great time this weekend getting to know it. I hope I can be of assistance in the future as well.
> 
> I've uploaded a new commit addressing your comments.

Thank you very much for willing to help, I am looking forward to your future PRs. Feel free to submit any issues/pr/discussion.
